TestSuite/integration.py

```diff
@@ -30,9 +35,15 @@ def __init__(
         self.create_unified = create_unified
 
         self.path = str(self._tmpdir_integration_path)
-        self.code = File(
-            self._tmpdir_integration_path / f"{self.name}.py", self._repo.path
-        )
+        self.type = _type
+        if self.type == "python":
+            self.code = File(
+                self._tmpdir_integration_path / f"{self.name}.py", self._repo.path
+            )
+        else:
```

Comment:
what about ps?

Reply:
I didn't need it in this PR, hence i didn't implement it.